app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
Specify the type for the attribute. 
HINT: You can use `from __future__ import annotation`